network-mux/src/Network/Mux/Types.hs

```diff
@@ -331,7 +331,7 @@ data MuxTrace ptcl =
     | MuxTraceRecvHeaderEnd (MuxSDU ptcl)
     | MuxTraceRecvPayloadStart Int
     | MuxTraceRecvPayloadEnd BL.ByteString
-    | MuxTraceRecvDeltaQueue (MuxSDU ptcl) Time
+    | MuxTraceRecvDeltaQObservation (MuxSDU ptcl) Time
```

Comment:
If we're using this for actual internal observation, rather than just logging etc, I think we should use a separate tracer. It makes me nervous using the same one.

Reply:
I had the same sort of feeling - we could start a new DeltaQ tracer now or leave this until the inclusion of the DeltaQ protocol exchange (when this trace would move to that component anyway)

Reply:
I would like to leave this as is for this initial version - noting that this should change into a new trace in the fullness of time.